battenberg/cli.py

```diff
@@ -102,5 +103,12 @@ def upgrade(ctx, **kwargs):
     try:
         battenberg = Battenberg(open_repository(ctx.obj['target']))
         battenberg.upgrade(**kwargs)
+    except MergeConflictException:
```

Comment:
thoughts on a test around this? should it exit w/ a non-0 exit code?

Reply:
Yeah I was being lazy by not adding a test, didn't look I bothered writing any tests for the CLI in the last round of development. I'll add some now :) 

I went back and forth about whether this is an error scenario or not. Typically this error will arise if the template changes in such a way that you expect to generate conflicts (ie. when making breaking changes) so in some ways it's a known state and should exit 0. I'm going to put in the exit 1 status though as I think if the `battenberg upgrade` command was ever chained then the next function likely wouldn't want to automatically execute as the only way to resolve this state is manually.

Reply:
Done in https://github.com/zillow/battenberg/pull/13/commits/d2891c2608c7d3b7e6f92e075aef9820534a7c07